src/main/java/com/spotify/reaper/storage/postgresql/UuidArgumentFactory.java

```diff
@@ -0,0 +1,35 @@
+package com.spotify.reaper.storage.postgresql;
+
+import org.skife.jdbi.v2.StatementContext;
+import org.skife.jdbi.v2.tweak.Argument;
+import org.skife.jdbi.v2.tweak.ArgumentFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.UUID;
+
+/**
+ * Provides JDBI a method to map UUID value to a long (most sig bits) value in database.
+ */
+public class UuidArgumentFactory implements ArgumentFactory<UUID> {
+
+  private static final Logger LOG = LoggerFactory.getLogger(UuidArgumentFactory.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)